UEBPrep/app/assets/javascripts/cards.js

```diff
@@ -0,0 +1,56 @@
+$(document).ready(function() {
+    $('#card_type').change(function(){
+        $('#card_title_tr').remove();
+        $('#card_textarea_tr').remove();
```

Comment:
This is great for a first implementation, perhaps in the future we can change this to just apply/unapply a class to the element that makes it visible, as opposed to dynamically changing the dom?